config/honeybadger.php

```diff
@@ -111,27 +114,10 @@
         'enabled' => true,
 
         /**
-         * Events which should automatically be recorded by the Honeybadger client.
+         * Events which should automatically be recorded by the Honeybadger client as breadcrumbs.
          * Note that to track redis events, you need to call `Redis::enableEvents()` in your app.
          */
-        'automatic' => [
-            Breadcrumbs\DatabaseQueryExecuted::class,
-            Breadcrumbs\DatabaseTransactionStarted::class,
-            Breadcrumbs\DatabaseTransactionCommitted::class,
-            Breadcrumbs\DatabaseTransactionRolledBack::class,
-            Breadcrumbs\CacheHit::class,
-            Breadcrumbs\CacheMiss::class,
-            Breadcrumbs\JobQueued::class,
-            Breadcrumbs\MailSending::class,
-            Breadcrumbs\MailSent::class,
-            Breadcrumbs\MessageLogged::class,
-            Breadcrumbs\NotificationSending::class,
-            Breadcrumbs\NotificationSent::class,
-            Breadcrumbs\NotificationFailed::class,
-            Breadcrumbs\RedisCommandExecuted::class,
-            Breadcrumbs\RouteMatched::class,
-            Breadcrumbs\ViewRendered::class,
-        ],
+        'automatic' => HoneybadgerLaravel::DEFAULT_EVENTS,
```

Comment:
This is an important change. I have extended the functionality of these events in order to use them both for breadcrumbs and the events api. During this change, I also modified their namespace (i.e. `Breadcrumbs\DatabaseQueryExecuted::class`  to `Events\ DatabaseQueryExecuted::class`). Since these classes are part of configuration, I kept the old ones and marked them as deprecated (as empty classes extending the new ones). In order to keep backwards compatibility, I check for the old namespace in 2 places:
- when [registering](https://github.com/honeybadger-io/honeybadger-laravel/pull/125/files#diff-74ffe2cea11eaa23d7fa43b69ba8489b02308ebff899f9bb63954311ba282c0cR190) the event handlers (I want to make sure we are not registering them twice; one with the old namespace and one with the new)
- when [checking](https://github.com/honeybadger-io/honeybadger-laravel/pull/125/files#diff-e86e3ed365dd30c7c16ec398ec6f2758992bdd37798bbf2d123d9c23ad369ea5R44) if the event should be reported, either as a breadcrumb or as an event